docs/content/users/quickstart.md

```diff
@@ -359,29 +359,38 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
 
 === "Magento 2"
 
-    Normal details of a Composer build for Magento 2 are on the [Magento 2 site](https://devdocs.magento.com/guides/v2.4/install-gde/composer.html). You must have a public and private key to install from Magento’s repository. When prompted for “username” and “password” in `composer create`, it’s asking for your public and private keys.
+    Normal details of a Composer build for Magento 2 are on the [Magento 2 site](https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/composer.html). You must have a public and private key to install from Magento’s repository. When prompted for “username” and “password” in `composer create`, it’s asking for your public key as "username" and private key as "password".
 
     ```bash
-    mkdir ddev-magento2 && cd ddev-magento2
-    ddev config --project-type=magento2 --php-version=8.1 --database=mariadb:10.6 --docroot=pub --disable-settings-management
+    SITENAME=ddev-magento2
+    mkdir -p ${SITENAME} && cd ${SITENAME}
+    ddev config --project-type=magento2 --php-version=8.2 --database=mariadb:10.6 --docroot=pub --disable-settings-management --upload-dirs=media
+    ddev config --web-environment-add=COMPOSER_HOME="/var/www/html/.ddev/homeadditions/.composer"
     ddev get ddev/ddev-elasticsearch
     ddev start
-    ddev composer create --repository=https://repo.magento.com/ magento/project-community-edition -y
+    ddev composer create --repository=https://repo.magento.com/ magento/project-community-edition --no-dev -y
```

Comment:
`--no-dev` is not needed here because people need dev dependencies to run things like phpunit tests.

```suggestion
    ddev composer create --repository=https://repo.magento.com/ magento/project-community-edition -y
```

Reply:
Unfortunately it creates such a huge vendor directory that it works against itself. I'm pretty sure people can sort out what to do.  I added that because we couldn't even upload the resultant code to GitHub release. However, your point is still valid.